core-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java

```diff
@@ -49,14 +51,16 @@
   private static final Logger LOG = Logger.getInstance(CloudDebuggerClient.class);
   private static final String ROOT_URL = "https://clouddebugger.googleapis.com";
   private static final ConcurrentHashMap<String, Debugger> myDebuggerClientsFromUserEmail =
-    new ConcurrentHashMap<String, Debugger>();
+      new ConcurrentHashMap<String, Debugger>();
+  private static final String USER_AGENT
```

Comment:
I think we should at least include plugin version as well as IDEA platform type and version in the user-agent, and use this spec for formatting http://www.ietf.org/rfc/rfc2616.txt